frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts

```diff
@@ -56,58 +81,141 @@ describe('Add/remove disks and NICs on respective VM pages', () => {
     await browser.wait(
       until.or(
         async () => {
-          return !(await $(networkTypeDropdownId).isEnabled());
+          return !(await $(networkTypeDropdownID).isEnabled());
         },
         async () => {
-          return !(await getDropdownOptions(networkTypeDropdownId)).includes(
-            networkInterface.networkDefinition,
+          return !(await getDropdownOptions(networkTypeDropdownID)).includes(
+            multusNetworkInterface.networkDefinition,
           );
         },
       ),
     );
   });
 });
+
+describe('NIC binding options', () => {
+  const bindingMethods = [
+    networkBindingMethods.bridge,
+    networkBindingMethods.masquerade,
+    networkBindingMethods.sriov,
+  ].sort();
+  const nonPodNetworkBindingMethods = [
+    networkBindingMethods.bridge,
+    networkBindingMethods.sriov,
+  ].sort();
+  const wizard = new Wizard();
+  const leakedResources = new Set<string>();
+
+  beforeAll(async () => {
+    createResource(multusNAD);
+  });
+
+  afterAll(async () => {
+    deleteResource(multusNAD);
+  });
+
+  it('Test NIC binding in VM Wizard', async () => {
+    const provisionSource = { method: 'Container', source: basicVMConfig.sourceContainer };
+
+    await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+    await isLoaded();
+    await wizard.openWizard();
+
+    await wizard.fillName(getRandStr(5));
+    await wizard.fillDescription(testName);
+    await wizard.selectProvisionSource(provisionSource);
+    await wizard.selectOperatingSystem(basicVMConfig.operatingSystem);
+    await wizard.selectFlavor(basicVMConfig.flavor);
+    await wizard.selectWorkloadProfile(basicVMConfig.workloadProfile);
+    await wizard.next();
+
+    let rowsCount = await wizardView.tableRowsCount();
+    // Default binding method for Pod Networking NIC is masquerade
+    expect(wizardView.tableCell(rowsCount, networkWizardTabCol.binding).getText()).toEqual(
+      networkBindingMethods.masquerade,
+    );
+
+    await wizardView.activateTableRow(rowsCount);
+    await click($(wizardView.tableElementID('binding', rowsCount.toString())));
+    let options = await getDropdownOptions(
+      wizardView.tableElementID('binding', rowsCount.toString()),
+    );
+    // All options are available for Pod Networking
+    expect(options.sort()).toEqual(bindingMethods);
+    await click(wizardView.cancelButton);
+
+    await wizard.addNIC(multusNetworkInterface);
+    rowsCount = await wizardView.tableRowsCount();
+    await wizardView.activateTableRow(rowsCount);
+    await click($(wizardView.tableElementID('binding', rowsCount.toString())));
+    options = await getDropdownOptions(wizardView.tableElementID('binding', rowsCount.toString()));
+    expect(options.sort()).toEqual(nonPodNetworkBindingMethods); // expect masquerade is not available option
+  });
+
+  xit('BZ(1753232) Check nic binding method in example VM', async () => {
```

Comment:
[Fix](https://github.com/openshift/console/pull/2919) for [BZ 1753232](https://bugzilla.redhat.com/1753232) is now merged, why `xit` here?

Also, nit: `NIC` :smiley: